dart/lib/src/hub.dart

```diff
@@ -0,0 +1,379 @@
+      );
+    } else if (message == null) {
+      _options.logger(
+        SeverityLevel.warning,
+        'captureMessage called with null parameter.',
+      );
+    } else {
+      final item = _stack.last;
+      if (item != null) {
+        try {
+          sentryId = await item.client.captureMessage(
+            message: message,
+            level: level,
+          );
+        } catch (err) {
+          _options.logger(
+            SeverityLevel.error,
+            'Error while capturing message with id: ${message}',
+          );
+        }
+      } else {
+        _options.logger(
+          SeverityLevel.fatal,
+          'Stack peek was null when captureMessage',
+        );
+      }
+    }
+    _lastEventId = sentryId;
+    return sentryId;
+  }
+
+  @override
+  void addBreadcrumb(Breadcrumb breadCrumb) {
+    // TODO: implement addBreadcrumb
+    throw UnimplementedError();
+  }
+
+  @override
+  void bindClient(SentryClient client) {
+    _stack.add(_StackItem(client, _stack.last.scope));
+  }
+
+  @override
+  void clearBreadcrumbs() {
+    // TODO: implement clearBreadcrumbs
+    throw UnimplementedError();
+  }
+
+  @override
+  Hub clone() {
+    // TODO: implement clone
+    throw UnimplementedError();
+  }
+
+  @override
+  void close() {
+    // TODO: implement close
+    throw UnimplementedError();
+  }
+
+  @override
+  void configureScope(ScopeCallback callback) {
+    // TODO: implement configureScope
+    throw UnimplementedError();
+  }
+
+  @override
+  void popScope() {
+    // TODO: implement popScope
+    throw UnimplementedError();
+  }
+
+  @override
+  void pushScope() {
+    // TODO: implement pushScope
+    throw UnimplementedError();
+  }
+
+  @override
+  void removeExtra(String key) {
+    // TODO: implement removeExtra
+    throw UnimplementedError();
+  }
+
+  @override
+  void removeTag({String key}) {
+    // TODO: implement removeTag
+    throw UnimplementedError();
+  }
+
+  @override
+  void setExtra(String key, String value) {
+    // TODO: implement setExtra
+    throw UnimplementedError();
+  }
+
+  @override
+  void setFingerPrint(List<String> fingerPrint) {
+    // TODO: implement setFingerPrint
+    throw UnimplementedError();
+  }
+
+  @override
+  void setLevel(SeverityLevel level) {
+    // TODO: implement setLevel
+    throw UnimplementedError();
+  }
+
+  @override
+  void setTag({String key, String value}) {
+    // TODO: implement setTag
+    throw UnimplementedError();
+  }
+
+  @override
+  void setTransaction(String transaction) {
+    // TODO: implement setTransaction
+    throw UnimplementedError();
+  }
+
+  @override
+  void setUser(User user) {
+    // TODO: implement setUser
+    throw UnimplementedError();
+  }
+
+  @override
+  void startSession() {
+    // TODO: implement startSession
+    throw UnimplementedError();
+  }
+
+  @override
+  void stopSession() {
+    // TODO: implement stopSession
+    throw UnimplementedError();
+  }
```

Comment:
https://github.com/getsentry/sentry-dart/pull/113/files#r505776960

Reply:
i removed both startSession & stopSession !?

Reply:
the comment was made on lines 229 to 287, so all of them